src/components/PastClients.js

```diff
@@ -5,50 +5,60 @@ import EmblaAutoScrollCarousel from "./embla/EmblaAutoScrollCarousel";
 const OPTIONS = { slidesToScroll: "auto", loop: true };
 
 const SLIDES = [
-  <StaticImage
-    className="w-full bg-walden-green"
-    imgStyle={{
-      objectFit: "scale-down",
-    }}
-    src="../images/client-logos/walden.svg"
-    alt="Walden Local"
-  />,
-  <StaticImage
-    className="w-full"
-    imgStyle={{
-      objectFit: "scale-down",
-    }}
-    src="../images/client-logos/glossier.svg"
-    alt="Glossier"
-  />,
-  <StaticImage
-    className="w-full"
-    imgStyle={{
-      objectFit: "scale-down",
-    }}
-    src="../images/client-logos/casper-logo.png"
-    alt="Casper"
-  />,
-  <StaticImage
-    className="w-full"
-    imgStyle={{
-      objectFit: "scale-down",
-    }}
-    src="../images/client-logos/florence-logo.png"
-    alt="Florence"
-  />,
-  <StaticImage
-    className="w-full"
-    imgStyle={{
-      objectFit: "scale-down",
-    }}
-    src="../images/client-logos/felcana-logo.png"
-    alt="Felcana"
-  />,
+  <div className="bg-walden-green rounded-2xl overflow-hidden drop-shadow-lg p-8">
```

Comment:
is there a way to make this & the glossier one generic so all the classes are the same? It will be helpful if we want to move to a model where we loop over them. One thing we can do is put the default background colour on the SVGs for each of them - I've done this in my PR for the walden SVG, and also updated it to a better version that is easier to work with.

Reply:
yeah i'll explore this